README.md

```diff
@@ -61,7 +61,32 @@ Dell Container Storage Modules Operator has been tested and qualified with
     * OpenShift Clusters 4.11, 4.12 with RHEL 8.x & RHCOS worker nodes
 
 ## Installation
-To install Dell Container Storage Modules Operator please refer the steps given here at [https://dell.github.io/csm-docs/docs/deployment/csmoperator/](https://dell.github.io/csm-docs/docs/deployment/csmoperator/)
+
+## Installation of Operators (Both OLM and Non OLM) in Dev Mode
+  1. Clone the repo: `git clone https://github.com/dell/csm-operator.git` 
+  2. Navigate one level inside the cloned repo: `cd csm-operator`
+  3. Execute `make install` to install the CRDs
+  4. Execute `make run` to have the operator running.
+  5. Install any of the operands (CSI Driver) using another session 
+
+NOTE: Closing the session where operator is running will stop the operator.
+
+## Installation of Operators using Scripts
```

Comment:
Does S have to be capitalized in scripts?

Reply:
Seems like if you capitalize Scripts then you should also capitalize the u in using. Thats the normal convention for headlines. Unless you chose to capitalize only the first letter of the phrase.

Reply:
Corrected